js/src/markdown-stream/markdown-stream.ts

```diff
@@ -167,7 +180,42 @@ class MarkdownElement extends LightElement {
   }
 
   #appendStreamingDot(): void {
-    this.lastElementChild?.appendChild(SVG_DOT)
+    this.#removeStreamingDot()
+
+    const findInnermost = (element: Element): Element => {
```

Comment:
I would love a high-level summary of what is function is trying to achieve. And if I follow correctly, it's to find the last `display: inline` element (so that we can append the dot inline next to that element)?

Reply:
The goal is to insert the streaming dot inside the last tag where we're currently streaming, which is sometimes contained inside other tags. It's not as direct as "find inline elements", because we don't want _any_ inline element, we want to figure out which element is being streamed into.

I need to tweak this a little bit to make sure we don't recurse into elements whose last child is a textNode (because that's where the streaming is happening).